src/movies.js

```diff
@@ -1,25 +1,89 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+
+
+function getAllDirectors(moviesArray) {
+
+ const getDir = moviesArray.map((el) => el.director)
+
+ return getDir
+}
+
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-function howManyMovies(moviesArray) {}
+
+function howManyMovies(moviesArray) {
+    
+    const dramaSteve = moviesArray.filter((el) => el.director === "Steven Spielberg" && el.genre.includes("Drama") === true)
```

Comment:
The `=== true` is not necessary after the `includes()` method because the `includes()` method already returns a boolean value indicating whether the element is present in the array or not.